src/main.js

```diff
@@ -1,75 +0,0 @@
-require('./main.less');
-require('../node_modules/leaflet/dist/leaflet.css');
-
-var $ = require('jquery'),
-	GlobalAddButton = require('./controls/global-add-button/'),
-	UploadDialog = require('./views/upload-dialog/'),
-	CodeManager = require('./views/code-manager/'),
-	Settings = require('./views/settings/'),
-	GeolocationViewer = require('./views/geolocation-viewer/'),
-	SideMenu = require('./views/side-menu/'),
-	ListView = require('./views/list-view/'),
-	mainTemplate = require('./main.vash'),
-	ready = require('dom-ready');
-	//locationWatch = require('./activity-location-watch');
-
-// function keepAppAlive(){
-// 	if (typeof cordova == 'undefined') return;
-
-// 	try {
-// 		cordova.plugins.backgroundMode.setDefaults({ 
-// 			title: 'Ethoinformatics',
-// 			text:'The app is in background mode.',
-// 			ticker:'Ethoinformatics is still running.',
-// 		});
-// 		cordova.plugins.backgroundMode.enable();
-// 	} catch (e){
-// 	}
-// }
-
-ready(function appLoad(){
```

Comment:
I'd like to go over this line-by-line to make sure I understand how the interface is built.